native/mod_proxy_cluster/mod_proxy_cluster.c

```diff
@@ -297,17 +377,22 @@ static apr_status_t create_worker(proxy_server_conf *conf, proxy_balancer *balan
                     if (proxyhctemplate != NULL) {
                         add_hcheck(server, conf, worker);
                     }
+                    ap_log_error(APLOG_MARK, APLOG_DEBUG, 0, server,
+                                 "Created: REUSING %s (scheme: %s hostname %s hostname_ex %s port %d route %s name %s) cleaning...",
+                                  url, worker->s->scheme, worker->s->hostname, worker->s->hostname_ex, worker->s->port, worker->s->route, worker->s->name_ex );
```

Comment:
Shouldn't this be within `#if MODULE_MAGIC_NUMBER_MAJOR == 20120211 && MODULE_MAGIC_NUMBER_MINOR >= 124`? If so, the same applies to those logs with `name_ex` below.

Reply:
Yes there are some more of the same :-(